examples/README.md

```diff
@@ -0,0 +1,61 @@
+# Examples for Testing Features Locally
+
+The example projects are meant to be used to test features locally by contributors working on this SDK. In the `requirements.txt` file, the deepgram package used is the local version:
+
+```
+-e ../../
```

Comment:
What is this code snippet referencing? Is there a package version in the `../..` directory somewhere?

Reply:
When I researched how we could use a local file for the deepgram package (instead of installing the package the normal way), I learned that we could add `-e ../../` to the requirements.txt file, creating the editable reference to the local deepgram package. 

The way I understand it is the -e flag indicates editable mode, meaning that the package is being installed from the directory in a way that any changes made to the code are immediately reflected without needing to reinstall. Unlike traditional package installations, which often specify specific versions, we're using the editable reference to allow us to work on and test the deepgram package directly within our project....

Now what I'm not sure is, based on your other comment, is this the best way to do this? We want to have folder like this in all the SDKs which has a pre-recorded and a streaming file separate from the SDK. The idea is to use it to test local changes. 

Do you want this to be changed to the method you suggested below where we do this:

```
pip uninstall deepgram-sdk  # If it's already installed
cd /path/to/deepgram-python-sdk/
pip install -e .
```

I want to do this in a way that makes sense to people so please let me know if you think I should just scrap this method

Reply:
Interesting, I didn't understand what the `-e ../../` was doing. So yeah, it's doing the same thing as `pip install -e .`. I don't have a strong preference for which way we tell users how to install their local version of the Deepgram SDK. If we keep the `-e ../../` in the `requirements.txt` file, can we add a comment in the Readme explaining that running `pip install requirements.txt` will install the package itself? Typically running `pip install requirements.txt` installs the _dependencies_ for the package rather than the package itself, so we'd just want to document what's going on since it's a little out of the ordinary (but still okay if you want to go that route imo).